api/controllers/UserController.js

```diff
@@ -367,6 +407,75 @@ module.exports = {
     }
   },
 
+  /*
+   * @api [get] /user
+   * tags:
+   *  - user
+   * summary: Returns all users the requesting user has access to.
+   * parameters:
+   *   - name: sort
+   *     description: An attribute to sort by
+   *     in: query
+   *     type: string
+   *     default: name
+   *   - name: offset
+   *     description: Number of users to offset
+   *     in: query
+   *     type: integer
+   *     default: 0
+   *   - name: limit
+   *     description: Maximum number of users to return
+   *     in: query
+   *     type: integer
+   *     default: 50
+   *   - name: fields
+   *     description: Fields to be returned (leave empty to return all)
```

Comment:
Maybe indicate the string format, ex: comma separated list of fields.

Reply:
You know, I realized while looking at this that the function shouldn't even exist in v3. We won't be providing the ability to look at another user. Even admins cannot do it anymore.

I'll remove the whole doc block and route.